generator/tests/init-sources/init-sources.spec.ts

```diff
@@ -65,6 +65,8 @@ describe("Test Extensions", () => {
 
     function initGit(path: string) {
         cp.execSync('git init', { cwd: path });
+        cp.execSync('git config --local user.name "test test"', { cwd: path });
+        cp.execSync('git config --local user.email test@gmail.com', { cwd: path });
```

Comment:
maybe better to use `example.com` or `eclipse.org` than `gmail` there

Reply:
Done

Reply:
@AndrienkoAleksandr I think email should be still with pattern something@something (like `user@example.com`)